bin/polyline.bin.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+var process = require('process');
```

Comment:
`process` is a global object: you shouldn't need to require it. `require('process')` throws in node v0.10.x